### PR TITLE
AAP-7610 Added troubleshooting topic and included procedure

### PR DESCRIPTION
**Pull request by dcdacosta**
This PR addresses the changes required by https://issues.redhat.com/browse/AAP-7610.

Changes include:

- Added a Troubleshooting topic
- Added procedure to recover when the CR name matches an existing resource
- Added a link within the Backup section pointing to troubleshooting

Preview link (VPN required): http://file.rdu.redhat.com/~ddacosta/AAP7610/master.html

Affects `/titles/aap-operator-backup`

### CODE REVIEW

**Review by rooftopcellist — APPROVED** [review]

File: downstream/modules/platform/proc-aap-controller-restore.adoc [hunk 1]
```diff
@@ -7,7 +7,7 @@ Use this procedure to restore a previous controller deployment from an Automatio
 
 [NOTE]
 ====
-The name specified for the new AutomationController custom resource must not match an existing deployment or the recovery process will fail.
+The name specified for the new AutomationController custom resource must not match an existing deployment or the recovery process will fail. Ifthe name specified does match an existing deployment, see xref:aap-troubleshoot-backup-recover[Troubleshooting] for steps to resolve the issue.
```
Comment (ariordan-redhat) [inline downstream/modules/platform/proc-aap-controller-restore.adoc:10]:
```suggestion
The name specified for the new AutomationController custom resource must not match an existing deployment or the recovery process will fail. 
If the name specified does match an existing deployment, see xref:aap-troubleshoot-backup-recover[Troubleshooting] for steps to resolve the issue.
```

**Review by ariordan-redhat — COMMENTED** [review]
Just a typo fix.
Otherwise, LGTM

**Comment by ariordan-redhat** [comment]
LGTM

**Review by ariordan-redhat — COMMENTED** [review]
LGTM

**Pull request merged**